ca_gazebo/launch/create_camera_world.launch

```diff
@@ -0,0 +1,32 @@
+<launch>
+  <param name="create1/amcl/initial_pose_x" value="0"/>
+  <param name="create1/amcl/initial_pose_y" value="4"/>
+  <param name="create1/amcl/initial_pose_a" value="0"/>
+  
+  <arg name="env" default="camera_world"/>
+
+  <arg name="paused" value="$(optenv PAUSED false)"/>
+  <arg name="gui" value="$(optenv GUI true)"/>
+  <arg name="debug" value="$(optenv DEBUG false)"/>
+  <env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
```

Comment:
This environment variable is not necessary. Do what we discussed in the meeting the other day:

- Delete this and see if it works.
- If not, you have to edit the `export` tag in `package.xml`.